src/main/java/com/sequenceiq/cloudbreak/service/stack/connector/ParallelCloudResourceManager.java

```diff
@@ -74,12 +73,12 @@
         try {
             Set<Resource> resourceSet = new HashSet<>();
             CloudPlatform cloudPlatform = stack.cloudPlatform();
-            stackUpdater.updateStackStatus(stack.getId(), Status.REQUESTED, "Creation of cluster infrastructure has been requested.");
+            stackUpdater.updateStackStatusReason(stack.getId(), "Creation of cluster infrastructure resources has been started.");
```

Comment:
This method does not trigger notification, so please use the 'updateStackStatus' method or fix the used method to trigger notification when necessary.